### ISSUE TITLE
UI doesn't show metrics.

### ISSUE THREAD

**Issue by jgordijn** [magneticio/vamp#435]
There are no metrics in the UI, even after running `ab -n 100 http://application_url:9050/dashboard`
See screenshot 
<img width="1906" alt="screen shot 2015-11-19 at 14 23 03" src="https://cloud.githubusercontent.com/assets/210142/11271974/d3d66c2c-8ec9-11e5-8251-8f8b1ef66398.png">

**Comment by jgordijn** [comment]
This is the logging I see in pulse:
```
10:54:15.798 | ERROR | io.vamp.common.notification.Notification |                                          | org.elasticsearch.common.util.concurrent.EsRejectedExecutionException: rejected execution (queue capacity 200) on org.elasticsearch.action.support.replication.TransportShardReplicationOperationAction$AsyncShardOperationAction$1@61a23e30 (of class org.elasticsearch.common.util.concurrent.EsRejectedExecutionException)
scala.MatchError: org.elasticsearch.common.util.concurrent.EsRejectedExecutionException: rejected execution (queue capacity 200) on org.elasticsearch.action.support.replication.TransportShardReplicationOperationAction$AsyncShardOperationAction$1@61a23e30 (of class org.elasticsearch.common.util.concurrent.EsRejectedExecutionException)
	at io.vamp.pulse.elasticsearch.ElasticsearchActor$$anonfun$io$vamp$pulse$elasticsearch$ElasticsearchActor$$insertEvent$1.applyOrElse(ElasticsearchActor.scala:116) ~[vamp-pulse.jar:0.7.11]
	at io.vamp.pulse.elasticsearch.ElasticsearchActor$$anonfun$io$vamp$pulse$elasticsearch$ElasticsearchActor$$insertEvent$1.applyOrElse(ElasticsearchActor.scala:115) ~[vamp-pulse.jar:0.7.11]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) ~[vamp-pulse.jar:0.7.11]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) ~[vamp-pulse.jar:0.7.11]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply$mcV$sp(BatchingExecutor.scala:91) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.BatchingExecutor$BlockableBatch$$anonfun$run$1.apply(BatchingExecutor.scala:91) [vamp-pulse.jar:0.7.11]
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:72) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:90) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40) [vamp-pulse.jar:0.7.11]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:397) [vamp-pulse.jar:0.7.11]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [vamp-pulse.jar:0.7.11]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [vamp-pulse.jar:0.7.11]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [vamp-pulse.jar:0.7.11]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [vamp-pulse.jar:0.7.11]
```

**Comment by jgordijn** [comment]
After restarting (in this order)
pulse
core
router

It start working. This is a machine that starts these services when the machine boots.

**Comment by jgordijn** [comment]
And now after running a few minutes it stopped working again.

**Comment by dragoslav** [comment]
#322 
In 0.8.0 it's changed how we collect/publish metrics and this particular issues should be solved.

**Comment by dragoslav** [comment]
fixed in 0.8.0
